CHANGELOG.md

```diff
@@ -0,0 +1,34 @@
+# Changelog
+
+Notable changes to this project are documented in this file. The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+Breaking changes (😱!!!):
+
+New features:
+
+Bugfixes:
+
+Other improvements:
+
+## [v0.4.0](https://github.com/purescript-contrib/purescript-matryoshka/releases/tag/v0.4.0) - 2018-06-28
+
+Updates for PS@0.12 @xgrommx
```

Comment:
Do you mind breezing through this file to add bullets to each of these sections? For example, in this case:

```suggestion
- Updates for PS@0.12 @xgrommx
```

I know it's a trivial thing, but that makes things a bit more consistent with the format we otherwise use. I've been doing this for other libraries.